File: 백준/Bronze/27855. Cornhole/Cornhole.py
import sys
input=lambda:sys.stdin.readline().rstrip()
s1=[*map(int,input().split())]
s2=[*map(int,input().split())]
t1=s1[0]*3+s1[1]*1
t2=s2[0]*3+s2[1]*1
if t1==t2:
    print("NO SCORE")
else:
    if t1>t2:
        print(1,t1-t2)
    else:
        print(2,t2-t1)

File: 백준/Bronze/27855. Cornhole/Cornhole.cc
#include <iostream>
#include <vector>
using namespace std;
int main(void) {
    int e1, e2;
    vector<int> t(2);
    for (int i = 0; i < 2; i++) {
        cin >> e1 >> e2;
        t[i] = 3 * e1 + 1 * e2;
    }
    if (t[0] == t[1])
        cout << "NO SCORE";
    else if (t[0] > t[1])
        cout << 1 << ' ' << t[0] - t[1];
    else
        cout << 2 << ' ' << t[1] - t[0];
    return 0;
}